pysat/_meta.py

```diff
@@ -609,10 +609,46 @@ def __eq__(self, other_meta):
 
         """
 
-        if isinstance(other_meta, Meta):
-            # Check if the variables and attributes are the same
-            for iter1, iter2 in [(self.keys(), other_meta.keys()),
-                                 (self.attrs(), other_meta.attrs())]:
+        if not isinstance(other_meta, Meta):
+            # The object being compared wasn't even the correct class
+            return NotImplemented
```

Comment:
Why is this a NotImplemented rather than a False?

Reply:
I added `NotImplemented`, as recommended in several custom `__eq__` tutorials.  This lets the second object attempt to define the equality before failure.